src/test/java/unit/DataCollectionReferenced.java

```diff
@@ -38,7 +42,7 @@ public class DataCollectionReferenced extends SingleWorkspaceUnit {
   private static final String DATA_COLLECTION_BUCKET_RECOLLECTION_NAME =
```

Comment:
Sorry, might have changed this by accident

DATA_COLLECTION_BUCKET_RESOURCE_NAME

Can you also update

DATA_COLLECTION_RECOLLECTION_NAME
DATA_COLLECTION_GIT_RECOLLECTION_NAME